tests/test_other.py

```diff
@@ -8916,6 +8916,21 @@ def test_wasm_sourcemap_dead(self):
     # has only two entries
     self.assertRegexpMatches(output, r'"mappings":\s*"[A-Za-z0-9+/]+,[A-Za-z0-9+/]+"')
 
+  def test_wasm_sourcemap_relative_paths(self):
+    filename = 'a.cpp'
+    shutil.copyfile(path_from_root('tests', 'hello_123.c'), 'a.cpp')
+    filenames = [
+      filename,
+      os.path.abspath(filename)
+    ]
+    if not WINDOWS:
```

Comment:
Why is this platform-specific?

Reply:
I don't think `./filename` is valid on windows?

Reply:
It is! Windows happily accepts forward-slashes (we discussed this when I made the change to unconditionally replace `\` with `/` too).

Reply:
Oh, even with the dot for the current dir?

Reply:
Yeah, sure, relative paths are pretty much the same everywhere (that is, `.` and `..` work in the same way).